security-ldap/build.gradle

```diff
@@ -1,4 +1,10 @@
+plugins {
+    id "io.micronaut.build.internal.module"
+}
+
 dependencies {
+    annotationProcessor "io.micronaut:micronaut-graal"
```

Comment:
I think that this should not be necessary. we were applying this in the root build to every submodule. Thus, I have keep it to avoid doing a breaking change accidentally.